packages/migrate/src/utils/setupPostgres.ts

```diff
@@ -13,23 +13,26 @@ export async function setupPostgres(options: SetupParams): Promise<void> {
   const { dirname } = options
   const credentials = uriToCredentials(connectionString)
 
-  let schema = `
-  SELECT 'CREATE DATABASE tests-migrate-shadowdb' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'tests-migrate-shadowdb');
-  SELECT 'CREATE DATABASE ${credentials.database}' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '${credentials.database}');
-  `
-  if (dirname !== '') {
-    schema += fs.readFileSync(path.join(dirname, 'setup.sql'), 'utf-8')
-  }
-
-  await createDatabase(connectionString).catch((e) => console.error(e))
-
-  const db = new Client({
-    connectionString: connectionString,
+  // Connect to default db
+  const dbDefault = new Client({
+    connectionString: connectionString.replace(credentials.database!, 'postgres'),
   })
+  await dbDefault.connect()
+  await dbDefault.query(`DROP DATABASE IF EXISTS "${credentials.database}-shadowdb";`)
```

Comment:
Why does PG need to be dropped, but not other DBs?

Reply:
It's my way of doing a create table if not exists which doesn't work on PostgreSQL 😢 
So it always starts clean